doc/source/ray-observability/ray-metrics.rst

```diff
@@ -75,16 +75,99 @@ Then go to to the location of the binary and run grafana using the built in conf
     ./bin/grafana-server --config /tmp/ray/session_latest/metrics/grafana/grafana.ini web
 
 Now, you can access grafana using the default grafana url, `http://localhost:3000`.
-If this is your first time, you can login with the username: `admin` and password `admin`.
+You can then see the default dashboard by going to dashboards -> manage -> Ray -> Default Dashboard. The same metric graphs are also accessible via :ref:`Ray Dashboard <ray-dashboard>`.
 
-.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard/grafana_login.png
-    :align: center
+.. tip::
 
-You can then see the default dashboard by going to dashboards -> manage -> Ray -> Default Dashboard. The same metric graphs are also accessible via :ref:`Ray Dashboard <ray-dashboard>`.
+  If this is your first time using Grafana, you can login with the username: `admin` and password `admin`.
 
-.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard/default_grafana_dashboard.png
+.. image:: images/graphs.png
     :align: center
 
+.. _system-metrics:
+
+System Metrics
+--------------
+Ray exports a number of system metrics, which provide introspection into the state of Ray workloads, as well as hardware utilization statistics. The following table describes the officially supported metrics:
+
+.. note::
+
+   Certain labels are common across all metrics, such as `SessionName` (uniquely identifies a Ray cluster instance), `instance` (per-node label applied by Prometheus, and `JobId` (Ray job id, as applicable).
+
+.. list-table:: Ray System Metrics
+   :header-rows: 1
+
+   * - Prometheus Metric
+     - Labels
+     - Description
+   * - `ray_tasks`
+     - `Name`, `State`
+     - Current number of tasks (both remote functions and actor calls) by state. The State label (e.g., RUNNING, FINISHED, FAILED) describes the state of the task. See rpc::TaskState for more information. The function/method name is available as the Name label.
+   * - `ray_actors`
+     - `Name`, `State`
+     - Current number of actors in a particular state. The State label is described by rpc::ActorTableData proto in gcs.proto. The actor class name is available in the Name label.
+   * - `ray_resources`
+     - `Name`, `State`
+     - Logical resource usage for each node of the cluster. Each resource has some quantity that is in USED state vs AVAILABLE state. The Name label defines the resource name (e.g., CPU, GPU).
+   * - `ray_object_store_memory`
+     - `Location`, `ObjectState`
+     - Object store memory usage in bytes, broken down by logical Location (SPILLED, IN_MEMORY, etc.), and ObjectState (UNSEALED, SEALED).
```

Comment:
Is the definition of them documented somewhere? I'd be nice if we have explanation for each state (SPILLED vs IN_MEMORY).